src/core/client/admin/routes/Moderate/ModerateSearchBar/Bar.tsx

```diff
@@ -39,13 +46,21 @@ const Bar: FunctionComponent<Props> = ({ title, options, onSearch }) => {
     ({ search }: { search: string }) => onSearch && onSearch(search),
     [onSearch]
   );
+  const searchInput = useRef<HTMLInputElement>(null);
   const blurOnEscProps = useBlurOnEsc(focused);
   const [
     mappedOptions,
     activeDescendant,
     keyboardNavigationHandlers,
   ] = useComboBox("moderate-searchBar-listBoxOption", options);
 
+  useEffect(() => {
+    key(HOTKEYS.SEARCH, () => {
+      if (searchInput && searchInput.current) {
+        searchInput.current.focus();
+      }
+    });
+  }, []);
```

Comment:
Because `useRef` will return the same object every render, but `searchInput.current` will return the current referenced object, you should probably tie this dep to the current ref:

```suggestion
  }, [searchInput.current]);
```